src/status_im/contexts/syncing/syncing_devices_list/view.cljs

```diff
@@ -26,19 +31,24 @@
 
 (defn view
   []
-  (let [devices                                     (rf/sub [:pairing/installations])
+  (let [theme                                       (quo.theme/use-theme)
+        devices                                     (rf/sub [:pairing/installations])
         devices-with-button                         (map #(assoc % :show-button? true) devices)
         user-device                                 (first devices-with-button)
         other-devices                               (rest devices-with-button)
         profile-color                               (rf/sub [:profile/customization-color])
         open-setup-syncing-with-customization-color (rn/use-callback (partial open-setup-syncing
                                                                               profile-color)
                                                                      [profile-color])
+        show-feature-unavailable-with-theme         (rn/use-callback (partial show-feature-unavailable
```

Comment:
A little nitpick @seanstrom:

I found this usage of `partial` and the other above it (not from this PR) confusing because `show-feature-unavailable` only expects one argument `theme` (but I had to check the function definition to be sure), thus there are no more args remaining from the partial application. Here `partial` is used only to generate an anonymous function, but this is quite uncommon in Clojure. If the implementation was `(rn/use-callback #(show-feature-available theme))` I would immediately know there was nothing else to understand about this callback.